test/src/de/dkfz/b080/co/knowledge/metadata/COMetadataAccessorSpec.groovy

```diff
@@ -1,13 +1,29 @@
 /*
- * Copyright (c) 2018 German Cancer Research Center (DKFZ).
+ * Copyright (c) 2018 German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ).
  *
- * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ * Distributed under the MIT License (license terms are at https://github.com/DKFZ-ODCF/COWorkflowsBasePlugin/LICENSE.txt).
  */
 package de.dkfz.b080.co.knowledge.metadata
 
-import spock.lang.Specification
+import de.dkfz.b080.co.common.BasicCOProjectsRuntimeService
+import de.dkfz.b080.co.files.Sample
+import de.dkfz.roddy.RoddyTestSpec
+import de.dkfz.roddy.config.ConfigurationConstants
+import de.dkfz.roddy.config.ConfigurationValue
+import de.dkfz.roddy.core.ExecutionContext
+import spock.lang.Shared
 
-class COMetadataAccessorSpec extends Specification {
+import static de.dkfz.b080.co.common.COConstants.CVALUE_ALLOW_SAMPLE_TERMINATION_WITH_INDEX
+import static de.dkfz.b080.co.common.COConstants.CVALUE_MATCH_EXACT_SAMPLE_NAMES
+import static de.dkfz.b080.co.common.COConstants.CVALUE_POSSIBLE_CONTROL_SAMPLE_NAME_PREFIXES
+import static de.dkfz.b080.co.common.COConstants.CVALUE_POSSIBLE_TUMOR_SAMPLE_NAME_PREFIXES
+import static de.dkfz.b080.co.common.COConstants.CVALUE_SELECT_SAMPLE_EXTRACTION_METHOD
+import static de.dkfz.b080.co.common.COConstants.CVALUE_USE_LOWER_CASE_FILENAMES_FOR_SAMPLE_EXTRACTION
+
+class COMetadataAccessorSpec extends RoddyTestSpec {
+
+    @Shared
+    static final ExecutionContext context = contextResource.createSimpleContext(COMetadataAccessorSpec)
```

Comment:
IO to test a simple parser. Try to make the code independent of ExecutionContext and Configuration.